elm-app/src/elm/Utils/Html.elm

```diff
@@ -40,6 +41,20 @@ showIf condition html =
         emptyNode
 
 
+{-| Conditionally show Html with a default Html if the condition doesn't pass
+A bit cleaner than using if expressions in middle of an html block:
+
+    showIfWithDefault True (text "I'm shown") (text "I'm default")
+
+-}
+showIfWithDefault : Bool -> Html msg -> Html msg -> Html msg
+showIfWithDefault condition html defaultHtml =
```

Comment:
I think we should switch to the existing package we have ([issue](https://github.com/Gizra/drupal-elm-starter/issues/151)) [viewMaybe](https://github.com/NoRedInk/view-extra/blob/81645c535adea1d08219395f28c9d803d55001b0/src/View/Extra.elm#L58)